src/field/mobile-features/index.md

```diff
@@ -2,144 +2,143 @@
 
-![GPS streaming mode to survey lines](./input-streaming-line.jpg "GPS streaming mode to survey lines")
+To stop the streaming mode, tap the **streaming** button and then **Stop streaming mode**.
 
-Once you are finished, tap **Done**. If you are capturing an area, the shape will be automatically closed by connecting the last and the first vertex.
+![Mergin Maps mobile app stop streaming mode](./mobile-streaming-mode-stop.jpg "Mergin Maps mobile app stop streaming mode")
 
-You can change how often you want to capture a vertex in **Line rec. interval** in [**Settings**](../mobile-app-ui/#streaming-mode-and-recording-settings). The interval can be defined as time elapsed in seconds or as distance travelled in metres.
+Once you finish surveying the feature, tap the **Record** button. If you are capturing an area, the shape will be automatically closed by connecting the last and the first vertex.
 
-:::tip
-The recording interval can be defined also in metres as distance travelled.
+:::tip Threshold intervals for streaming mode
+It is possible to set the **Threshold interval**, i.e. how often you want to capture a vertex when streaming, in [**Settings**](../mobile-app-ui/#streaming-mode-and-recording-settings) of the <MobileAppNameShort />. The interval can be defined as *time elapsed* in seconds or as *distance travelled* in metres.
 :::
 
-### Add non-spatial features
-[Non-spatial features](../../layer/non-spatial-data/) can also be added using <MobileAppName />. If you are using a non-spatial table for [value relations](../../layer/form-widgets/#value-relation) you can add and edit values in the app.
+## Editing features
+Features can be browsed, edited and deleted through the [Layers](../layers/) panel in <MobileAppName />.
 
-1. Tap the **More** button and navigate to **Layers**
-   ![Mergin Maps mobile app Layers panel](./input-layers.jpg "Mergin Maps mobile app Layers panel")
+Tap the **Layers** button in the bottom navigation panel and select a layer to see the list of features it contains. 
 
-2. Choose the non-spatial layer and tap **Add feature**
-   ![](./input-add-non-spatial-feature.jpg)
-   
-3. Fill in the attributes.
+![Layers in Mergin Maps mobile app](./mobile-layers-browse-features.jpg "Layers in Mergin Maps mobile app")
 
-## Editing features
-Spatial and non-spatial features can be browsed, edited and deleted in the [Layers](../layers/) panel in <MobileAppName />. Tap on the **Layers** in the **More** tab and select a layer to see the features it contains. 
-![Mergin Maps mobile app Layers panel](./input-layers.jpg "Mergin Maps mobile app Layers panel")
+To edit the attributes or geometry of a feature, select it from the list in the **Layers** panel. It is also possible to simply tap a feature on the map.
+
+Use the **Edit** button to open the attributes form. Here you can change the values of attributes as needed. To edit the geometry of a feature, tap the **Edit geometry** button.
```

Comment:
This "Edit" button is there only in case the layer is not marked as "read-only". Not sure if it is important to state it here.

Reply:
Good point, it will be useful to mention it in this section that read-only layers cannot be edited. Thanks!